src/index.css

```diff
@@ -64,6 +64,11 @@ input {
   background-color: #1f8655;
 }
 
+
```

Comment:
lets delete this big block

Reply:
I've been using the format document option from ctrl + shft + p to format, no red lines currently.